### PR TITLE
Added a notification functionality

### PR DESCRIPTION
**Pull request by bhavanmodha**
added a constant in const.py with notification number 7, and added an case in the is_notification_needed in the opensyslog_syslog.py

### CODE REVIEW

File: config.yaml.example [hunk 1]
```diff
@@ -8,16 +8,16 @@ logs:
...
+  default_notify_type: 6 # default:6, 1=Do not notify, 2=Notify only first time connect, 3=Notify only first time connect or IP changed, 4=always notify, 5=always notify till max count, 6=always notify till max count and then intermittent, 7=do not notify if device present in lookup file
   max_notify_count_per_device_per_day: 5 # default: 5, this will apply for all notification types, to disable it make this number very high like 99999
   do_not_disturb_start_hour: 22 # default: 22, if start/end both same then igored and will notify always
   do_not_disturb_end_hour: 8 # default: 8, if start/end both same then igored and will notify always, if end_hour is lower than start_hour it will assume next day!
 
 telegram:
-  chat_id: CHAT_ID # telegram chat-id something like 123456789
-  api_token: "API_TOKEN" # telegram api token something like 123456789:XYZ-ABCDEFGHIJKL
+  chat_id: # telegram chat-id something like 123456789
+  api_token: # telegram api token something like 123456789:XYZ-ABCDEFGHIJKL
```
Comment (scadaguru) [inline config.yaml.example]: These two lines above changed by removing "example data" and I don't see why it is required so please remove these two lines changes which is not necessary! Thanks

File: opensyslog_syslog.py [hunk 2]
```diff
@@ -142,6 +142,10 @@ def is_notification_needed(self, mac_address, ip_address, first_time_seen):
             case const.NOTIFY_CONNECT_EACH_TIME_WITH_MAX_PER_DAY_WITH_INTERMITTENT:
                 return (self.dhcp_ack_json[mac_address]["reconnect_count_per_day"] <= self.max_notify_count_per_device_per_day) or \
                     self.dhcp_ack_json[mac_address]["reconnect_count_per_day"] % 10 == 1
+            case const.NOTIFY_CONNECT_DEVICE_NOT_IN_LOOKUP_FILE:
+                if self.helper.lookup_device_name_from_csv(mac_address):
+                    return False
+                return True
```
Comment (scadaguru) [inline opensyslog_syslog.py]: These 3 lines should be simplified to just one line to make it more readable please!
```
return not self.helper.lookup_device_name_from_csv(mac_address)
```

**Review by scadaguru — CHANGES_REQUESTED** [review]
Appreciate your PR to make contribution to this project, thanks!
I have couple comments please address them before this PR can be merged!

File: config.yaml.example (hunk 1 above)
Comment (bhavanmodha) [reply config.yaml.example]: Sorry about that I might have changed it while testing, but I will be more careful. I have made the changes

File: opensyslog_syslog.py (hunk 2 above)
Comment (bhavanmodha) [reply opensyslog_syslog.py]: Yes, I totally agree with that. I have made the changes.

**Review by scadaguru — APPROVED** [review]
Looks good! thanks

**Pull request merged**